cpp/unreal_projects/SpearSim/Config/DefaultEngine.ini

```diff
@@ -54,7 +54,7 @@ r.DefaultFeature.AutoExposure.ExtendDefaultLuminanceRange=True
 ; Set the default map and game mode
 [/Script/EngineSettings.GameMapsSettings]
 GameDefaultMap=/Game/Scenes/debug_0000/Maps/debug_0000.debug_0000
-EditorStartupMap=/Game/Scenes/debug_0000/Maps/debug_0000.debug_0000
```

Comment:
Revert this change. I like having `debug_0000` as the default startup map for internal development. We will make `apartment_0000` the default map for external release builds, but that happens in a separate part of the code.